File: material/src/main/java/org/odk/hedera/material/ContextUtils.java
package org.odk.hedera.material;

import android.content.Context;
import android.util.TypedValue;

import androidx.annotation.AttrRes;

class ContextUtils {

    static int getAttributeValue(Context context, @AttrRes int resId) {
        TypedValue outValue = new TypedValue();
        context.getTheme().resolveAttribute(resId, outValue, true);
        return outValue.data;
    }
}

File: androidshared/src/main/java/org/odk/collect/androidshared/utils/ContextUtils.kt
package org.odk.collect.androidshared.utils

import android.content.Context
import android.util.TypedValue
import androidx.annotation.AttrRes

object ContextUtils {

    @JvmStatic
    fun getThemeAttributeValue(context: Context, @AttrRes resId: Int): Int {
        val outValue = TypedValue()
        context.theme.resolveAttribute(resId, outValue, true)
        return outValue.data
    }
}
